eudplib/ctrlstru/loopblock.py

```diff
@@ -86,7 +86,7 @@ def EUDLoopRange(start, end=None) -> Iterator[c.EUDVariable]:  # noqa: N802
 
     v = c.EUDVariable()
     v << start
-    if EUDWhile()(v < end):
+    if EUDWhile()(v >= end, neg = True): // EUDVariable(0) < EUDVariable(0) is always true
```

Comment:
Just a small nitpick:
```suggestion
    if EUDWhileNot()(v >= end):  # FIXME: EUDVariable(0) < EUDVariable(0) is always True
```
1. Line comment syntax is `# comment` in Python, which is different from epScript (`// comment`).
2. Albeit `EUDWhile()(conditions, neg=True)` and `EUDWhileNot()(conditions)` are same things, `EUDWhileNot` is preferred for readability.